design/sketch.webidl

```diff
@@ -715,7 +715,6 @@ dictionary GPUCommandEncoderDescriptor {
 // Fence
 
 dictionary GPUFenceDescriptor {
-    GPUQueue signalQueue = null;
```

Comment:
In this (new) proposal, what is the equivalent of creating a fence and setting a null `signalQueue` in `GPUFenceDescriptor`?

Reply:
The idea was that without multi-queue it would default to the default queue. But that's no longer relevant since the application will now create the fence from the default queue.